query-engine/connector-test-kit/src/test/scala/writes/ids/relation_pks/SinglePKRelationFieldSpec.scala

```diff
@@ -14,225 +14,6 @@ import util._
-    val project = ProjectDsl.fromString {
-      s"""
-         |model Parent {
-         |  name     String
-         |  age      Int
-         |  child_id Int @id
-         |
-         |  child Child  @relation(fields: [child_id], references: [id])
-         |}
-         |
-         |model Child {
-         |  id      Int    @id
-         |  name    String
-         |  parent  Parent
-         |}
-       """
-    }
-    database.setup(project)
-
-    val res1 = server.query(
-      """
-        |mutation {
-        |  createParent(data: { name: "Paul" , age: 40, child: { create: { id: 1, name: "Panther" }}}) {
-        |    name
-        |    age
-        |    child{
-        |       id
-        |       name
-        |    }
-        |  }
-        |}
-      """,
-      project
-    )
-
-    res1.toString() should be("{\"data\":{\"createParent\":{\"name\":\"Paul\",\"age\":40,\"child\":{\"id\":1,\"name\":\"Panther\"}}}}")
-
-    val res2 = server.query(
-      """
-        |mutation {
-        |  updateParent(where: { child_id: 1 } data: { age: { set: 41 }}) {
-        |    name
-        |    age
-        |  }
-        |}
-      """,
-      project
-    )
-
-    res2.toString() should be("{\"data\":{\"updateParent\":{\"name\":\"Paul\",\"age\":41}}}")
-
-    val res3 = server.query(
-      """
-        |mutation {
-        |  updateChild(where: { id: 1 } data: { parent: { update: { age: { set: 42 }}}}) {
-        |    parent { age }
-        |  }
-        |}
-      """,
-      project
-    )
-
-    res3.toString() should be("{\"data\":{\"updateChild\":{\"parent\":{\"age\":42}}}}")
-
-    val res4 = server.query(
-      """
-        |mutation {
-        |  upsertParent(
-        |    where: { child_id: 2 }
-        |    update: { age: { set: 43 }}
-        |    create: { name: "Milutin", age: 42, child: { create: { id: 2, name: "Nikola" } } }
-        |  ) {
-        |    age
-        |    child {
-        |      id
-        |    }
-        |  }
-        |}
-        |
-      """,
-      project
-    )
-
-    res4.toString() should be("{\"data\":{\"upsertParent\":{\"age\":42,\"child\":{\"id\":2}}}}")
-  }
-
-  // Mutations in this test:
-  //  create         | root   | checked
-  //  update         | root   | checked
-  //  delete         | root   | not possible (1!:1!)
-  //  upsert         | root   | checked
-  //  updateMany     | root   | unnecessary
-  //  deleteMany     | root   | unnecessary
-  //  nested create  | create | checked
-  //  nested update  | update | checked
-  //  nested connect | -      | not possible (1!:1!)
-  //  nested disconn | -      | not possible (1!:1!)
-  //  nested delete  | -      | not possible (1!:1!)
-  //  nested set     | -      | not possible (1!:1!)
-  //  nested upsert  | -      | not possible (1!:1!)
-  //  nested deleteM | -      | not possible (1!:1!)
-  //  nested updateM | -      | not possible (1!:1!)
-  "Using an ID that is also a 1!:1! multi-field relation" should "work" in {
```

Comment:
same as above. Should probably bring it back.

Reply:
There are 1:1! cases in the file covering this.